src/Object.h

```diff
@@ -110,7 +110,7 @@ class Object: public osg::PositionAttitudeTransform, public Clickable{
     void switchCullMask();
     void xorCullMask(unsigned int mask);
 
-    void displayName(float font_size = 0.1f);
+    void displayName(const float &font_size = 0.1f, const osg::Vec3 &pos = osg::Vec3(0,0,0), const osg::Vec4 &color = osg::Vec4(0,0,0,1));
```

Comment:
There are only three usages of `displayname()` outside of osgViz, all of them easy to fix. Imo it would be ok to break backwards compatibility here and avoid the default parameters.

Reply:
What is the bad thing about defaults? I'm just curious